src/main/java/n1/variables/operators/statements/expressions/Task1.java

```diff
@@ -0,0 +1,56 @@
+package n1.variables.operators.statements.expressions;
+
+import java.util.Scanner;
+
+public class Task1 {
+
+    public static void main(String[] args) {
+        System.out.println("Enter a number and click Enter:");
+        Scanner scaner = new Scanner(System.in);
+        String switchCondition = "";
```

Comment:
don't set default value, it should be enough to declare variable without value.

Reply:
doesn't work without initialization

Reply:
Works perfectly, it seems that you skipped some statement.